backend/src/nodes/nodes/image_utility/convolve.py

```diff
@@ -0,0 +1,70 @@
+from __future__ import annotations
+
+import cv2
+import numpy as np
+
+from . import category as ImageUtilityCategory
+from ...node_base import NodeBase
+from ...node_factory import NodeFactory
+from ...properties.inputs import ImageInput, NumberInput, NoteTextAreaInput
+from ...properties.outputs import ImageOutput
+
+
+@NodeFactory.register("chainner:image:image_convolve")
+class ImageConvolveNode(NodeBase):
+    def __init__(self):
+        super().__init__()
+        self.description = "Convolves input image with input kernel"
+        self.inputs = [
+            ImageInput("Image"),
+            NoteTextAreaInput("Kernel String"),
+            NumberInput("Padding", minimum=0, default=0),
+        ]
+        self.outputs = [
+            ImageOutput(
+                image_type="""
+                    let w = Input0.width;
+                    let h = Input0.height;
+
+                    let kernel = Input1;
+
+                    let padding = Input2;
+
+                    Image {
+                        width: int & floor((w + padding * 2 - 1) + 1),
+                        height: int & floor((h + padding * 2 - 1)+ 1),
```

Comment:
Since stride is gone, this can be simplified.

```suggestion
                        width: w + padding * 2,
                        height: h + padding * 2,
```